lib/rmagick_internal.rb

```diff
@@ -50,6 +51,11 @@ def trace_proc=(p)
         @trace_proc = p
       end
     end
+
+    def lib_version
+      @lib_version ||= Gem::Version.new(Magick::Magick_version.split[1].split('-').first)
```

Comment:
It's fairly conventional in Ruby to have versions as strings, so I'd probably use the `Gem::Version` bit below where we need it to compare.